cli/lib/kontena/cli/cloud/login_command.rb

```diff
@@ -0,0 +1,130 @@
+require 'uri'
+
+module Kontena::Cli::Cloud
+  class LoginCommand < Kontena::Command
+    include Kontena::Cli::Common
+
+    option ['-t', '--token'], '[TOKEN]', 'Use a pre-generated access token'
+    option ['-c', '--code'], '[CODE]', 'Use an authorization code'
+    option ['-v', '--verbose'], :flag, 'Increase output verbosity'
+
+    def execute
```

Comment:
The master login and cloud login commands have a lot of commonalities that should be extracted into some  AuthCommon thing.